application/controllers/registration.php

```diff
@@ -42,8 +43,12 @@ public function individual() {
      * This method let the users register teams.
      */
     public function team() {
+        $data['male'] = $this->people->get_male_athlete_from_school(
+            $this->session->userdata('id'));
```

Comment:
Shall we assign `$this->session->userdata('id')` to a new variable to avoid calling session twice?